### PR TITLE
[docs] Add changelog for v0.19.1

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @AndreyBelym @miherlosev

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v0.19.1 (2018-3-13)
+
+### Backward compatibility with the legacy test syntax has been restored ([#2210](https://github.com/DevExpress/testcafe/issues/2210))
+
+### Bug Fixes
+
+* The `document.all` property is now overridden ([testcafe-hammerhead/#1046](https://github.com/DevExpress/testcafe-hammerhead/issues/1046))
+* The `async` expression generator was added to *esotope* ([testcafe-hammerhead/#1510](https://github.com/DevExpress/testcafe-hammerhead/issues/1510))
```
Comment (miherlosev) [inline CHANGELOG.md]: Сloser to the truth:
Support proxying properties in async class methods (testcafe-hammerhead/#1510)
Fix wrong proxing of a `localStorage` check in WebWorkers (testcafe-hammerhead/#1510)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+
+### Backward compatibility with the legacy test syntax has been restored ([#2210](https://github.com/DevExpress/testcafe/issues/2210))
+
+### Bug Fixes
+
+* The `document.all` property is now overridden ([testcafe-hammerhead/#1046](https://github.com/DevExpress/testcafe-hammerhead/issues/1046))
+* Proxying properties in `async` class methods is now supported ([testcafe-hammerhead/#1510](https://github.com/DevExpress/testcafe-hammerhead/issues/1510))
+* Fixed wrong proxying of a `localStorage` check in WebWorkers ([testcafe-hammerhead/#1510](https://github.com/DevExpress/testcafe-hammerhead/issues/1510))
+* Function wrappers no longer break *is-defined* checks ([testcafe-hammerhead/#1496](https://github.com/DevExpress/testcafe-hammerhead/issues/1496))
+
```
Comment (miherlosev) [inline CHANGELOG.md]: Dublicate for testcafe-hammerhead/#1510.

The final changelog:
* The `document.all` property is now overridden ([testcafe-hammerhead/#1046](https://github.com/DevExpress/testcafe-hammerhead/issues/1046))
* Proxying properties in `async` class methods is now supported ([testcafe-hammerhead/#1510](https://github.com/DevExpress/testcafe-hammerhead/issues/1510))
* Fixed wrong proxying of a `localStorage` check in WebWorkers ([testcafe-hammerhead/#1496](https://github.com/DevExpress/testcafe-hammerhead/issues/1496))

Comment (VasilyStrelyaev) [reply CHANGELOG.md]: Oh, I thought you duplicated them deliberately https://github.com/DevExpress/testcafe/pull/2211#discussion_r174173455

Comment (miherlosev) [reply CHANGELOG.md]: Yes, it's my mistake.

**Pull request merged**